_episodes/08-conflict.md

```diff
@@ -0,0 +1,338 @@
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+I'll be able to get 40 extra minutes of beauty rest
+~~~
+{: .output}
+
+and commit that change to the `master` branch
+
+~~~
+$ git add mars.txt
+$ git commit -m "Add a line about the daylight on Mars."
+~~~
+{: .bash}
+
+~~~
+[master 5ae9631] Add a line in our home copy
+ 1 file changed, 1 insertion(+)
+~~~
+{: .output}
+
+Now that we're done with that, let's get to work on our comments about
+the temperature in the `marsTemp` branch.
+
+~~~
+$ git checkout marsTemp
+$ git branch
+~~~
+{: .bash}
+
+~~~
+* marsTemp
+  master
+~~~
+{: .output}
+
+Let's make a note in `mars.txt` about the temperature
+
+~~~
+$ nano mars.txt
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+Yeti will appreciate the cold
+~~~
+{: .output}
+
+Now let's commit this change to the `marsTemp` branch
+
+~~~
+$ git add mars.txt
+$ git commit -m "Add a line about the temperature on Mars"
+~~~
+{: .bash}
+
+~~~
+[master 07ebc69] Add a line in my copy
+ 1 file changed, 1 insertion(+)
+~~~
+{: .output}
+
+Great, now that we've added changes about the temperature
+we can merge them into the `master` branch. First, let's checkout the
+`master` branch.
+
+~~~
+$ git checkout master
+$ git branch
+~~~
+{: .bash}
+
+~~~
+  marsTemp
+* master
+~~~
+{: .output}
+
+And then merge the changes from `marsTemp`
+
+~~~
+$ git merge marsTemp
+~~~
+{: .bash}
+
+~~~
+On branch master
+You have unmerged paths.
+  (fix conflicts and run "git commit")
+
+Unmerged paths:
+  (use "git add <file>..." to mark resolution)
+
+  both modified:   mars.txt
+
+no changes added to commit (use "git add" and/or "git commit -a")
+
+~~~
+{: .output}
+
+![The Conflicting Changes](../fig/conflict.svg)
+
+Git detects that the changes made in one copy overlap with those made
+in the other and stops us from trampling on our previous work. It also
+marks that conflict in the affected file:
+
+~~~
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+<<<<<<< HEAD
+I'll be able to get 40 extra minutes of beauty rest
+=======
+Yeti will appreciate the cold
+>>>>>>> dabb4c8c450e8475aee9b14b4383acc99f42af1d
+~~~
+{: .output}
+
+Our change—the one in `HEAD`—is preceded by `<<<<<<<`.
+Git has then inserted `=======` as a separator between the conflicting changes
+and marked the end of the content downloaded from GitHub with `>>>>>>>`.
+(The string of letters and digits after that marker
+identifies the commit we've just downloaded.)
+
+It is now up to us to edit this file to remove these markers
+and reconcile the changes.
+We can do anything we want: keep the change made in the local repository, keep
```

Comment:
options should now be, keep the change made in `master`, keep change made in `tempMars`, keep both, or delete both